animation_nodes/nodes/color/set_vertex_color.py

```diff
@@ -1,62 +1,100 @@
 import bpy
-import numpy
 from bpy.props import *
 from ... events import propertyChanged
-from ... base_types import AnimationNode
+from ... base_types import AnimationNode, VectorizedSocket
+from ... data_structures import Color, ColorList, VirtualColorList
+from . c_utils import getLoopColorsFromVertexColors, getLoopColorsFromPolygonColors
+
+colorModeItems = [
+    ("LOOP", "Loop", "Set color of every loop vertex", "NONE", 0),
+    ("VERTEX", "Vertex", "Set color of every vertex", "NONE", 1),
+    ("POLYGON", "Polygon", "Set color of every polgon", "NONE", 2)    
+]
+
+colorLayerIdentifierTypeItems = [
+    ("INDEX", "Index", "Get vertex color layer based on the index", "NONE", 0),
```

Comment:
Remove index mode.

Reply:
Omar, I think `Index` mode is handier than the `Name` mode to get the `Vertex Color Layer`, is it not? The `Set Vertex Weight` and `Vertex Group Input` nodes also have this mode.

Reply:
Oh, my bad, I confused it with the other mode type. This can be left as is then.

Reply:
It is fine :)